app/src/test/java/dev/arkbuilders/navigator/data/utils/PathExtKtTest.kt

```diff
@@ -0,0 +1,88 @@
+package dev.arkbuilders.navigator.data.utils
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import java.nio.file.Path
+import java.nio.file.Paths
+import kotlin.io.path.name
+import kotlin.io.path.notExists
+
+@RunWith(MockitoJUnitRunner::class)
+class PathExtKtTest {
+
+    private lateinit var testee: Path
+
+    @BeforeEach
+    fun setUp() {
+        testee = Paths.get("notAndroid")
```

Comment:
But what does `notAndroid` mean?

Reply:
I would like to use `PATH` instead. It's more meaningful